README.md

```diff
@@ -14,18 +14,37 @@ ImGui is designed to enable fast iteration and empower programmers to create con
 
 ImGui is particularly suited to integration in realtime 3D applications, fullscreen applications, embedded applications, games, or any applications on consoles platforms where operating system features are non-standard.
 
-#####Namespace
-For ease of use I added a namespace alias for ImGui, feel free to disable it by defining CINDER_IMGUI_NO_NAMESPACE_ALIAS
+### Getting Started
+Download or check out this repository into your `Cinder/blocks` directory, then use Tinderbox to create a new project using ImGui.
 
-#####Initialization
-This is the most basic initialization:
+Once the project is created you'll need to include the header in your `.cpp` file:
+```c++
+#include "CinderImGui.h"
+```
+
+Initialize ImGui at setup:
 ```c++
 void CinderApp::setup()
 {
-    ui::initialize();
+    ImGui::initialize();
 }
 ```
-You can provide an ui::Options object to the initialize method to setup the ui the way you want:
+
+Then you're ready to start drawing components:
+```c++
+void CinderApp::draw()
+{
+    gl::clear( Color( 0, 0, 0 ) );
+    ImGui::Text("Hello, world!");
```

Comment:
I used the non-namespaces version to make it clearer that it's the full ImGui and that you can paste code in and not need to use the namespace alias.